apps/web/src/components/proofreading-progress.tsx

```diff
@@ -182,97 +198,203 @@ export const ProofreadingProgress = ({
   const contributorsLength = proofreadingData.contributors.length;
 
   return (
-    <div
-      className={cn(
-        'group p-2.5 hover:p-5 rounded-[20px] shadow-course-navigation justify-start items-start gap-2.5 inline-flex absolute right-6 top-4',
-        'max-md:hidden', //proofreadingData.isOriginalLanguage ? 'hidden' : 'max-md:hidden',
-        mode === 'dark'
-          ? 'bg-newBlack-3'
-          : contributorsLength > 2
-            ? 'bg-brightGreen-1'
-            : 'bg-darkOrange-0',
-      )}
-    >
-      <div className="group-hover:hidden flex items-center gap-1.5">
-        <SmallProgressImage progress={contributorsLength} />
-        <span
-          className={cn(
-            'max-lg:hidden body-12px-medium w-[118px] whitespace-pre-line',
-            mode === 'dark' ? 'text-white' : 'text-black',
-          )}
-        >
-          {contributorsLength === 3
-            ? t('proofreading.completed')
-            : t('proofreading.inProgress')}
-        </span>
-      </div>
-      <div className="hidden group-hover:flex max-w-[258px] flex-col gap-4 justify-center">
-        <div className="flex items-center gap-4">
-          <span
+    <div>
+      <Dialog>
```

Comment:
Background behind dialog gets too dark, may need to change it according to Figma (validate with Muriel)